Models/CubeVisibility/README.body.md

```diff
@@ -0,0 +1,11 @@
+This model demonstrates the `KHR_node_visibility` extension and its interaction with `KHR_animation_pointer`.
+
+- If your implementation shows either of the red cubes, it is not compliant with `KHR_node_visibility`.
+
+- If your implementation does not have the blue cube hiding and showing every 0.5 seconds, either the animation is not playing, or it does not support using `KHR_animation_pointer` to animate the `KHR_node_visibility` visible property.
+
+Animated screenshot of the test model in action:
+
+![Cube Visibility](screenshot/screenshot_animated.webp)
+
+[Show](https://gltf-interactivity.needle.tools?model=https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Assets/main/Models/CubeVisibility/glTF-Binary/CubeVisibility.glb) – [Download GLB](https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Assets/main/Models/CubeVisibility/glTF-Binary/CubeVisibility.glb)
```

Comment:
The `README.md` file will be autogenerated with these links; the `README.body.md` should not include them; same for the light visibility asset.